.magento.app.yaml

```diff
@@ -78,7 +78,7 @@ mounts:
 hooks:
     # We run build hooks before your application has been packaged.
     build: |
-        php ./vendor/bin/ece-tools build:generate
+        php ./vendor/bin/ece-tools build:generate &&
```

Comment:
Is there a reason we chose to use `&&` instead of `set -e`?  That seems like it would be cleaner and also take care of what was noted in MAGECLOUD-1563.